web/src/hooks/UseTags.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from "@tanstack/react-query";
+import { createTag } from "src/resources/TagResource";
+
+export function useTags(org: string, repo: string){
+    const {
+      mutate: mutateCreateTag,
+      isSuccess: successCreateTag,
+      isError: errorCreateTag,
+    } = useMutation(
+      async ({tag, manifest}: {tag: string; manifest: string}) =>
+        createTag(org, repo, tag, manifest),
+      {},
```

Comment:
why is this needed?

Reply:
It isn't really, just left over. It can be removed.